plugin/src/main/java/com/denizenscript/denizen/objects/properties/item/ItemAttributeModifiers.java

```diff
@@ -172,13 +183,4 @@ public static void register() {
             prop.setItemMeta(meta);
         });
     }
-
-    public static String getIdString(AttributeModifier modifier) {
```

Comment:
imo unifying to strings is cleaner/more intercompatible, vs. the multi-data-type ternary hacks you replaced it with

Reply:
There might be a cleaner way to implement it (split it into 2 `if` checks or something), but the problem is the defaulting to `minecraft:` namespace Denizen has for `NamespacedKey` inputs - could maybe try concatenating `"minecraft:"` and having two checks or something? but that'd be worse than just comparing the key/UUID imo.